src/mystery_gift.c

```diff
@@ -0,0 +1,293 @@
+#include "global.h"
+#include "mystery_gift.h"
+#include "save.h"
+#include "save_arrays.h"
+#include "constants/save_arrays.h"
+
+BOOL MysteryGiftTagIsValid(u32 tag);
+MysteryGift* SaveMysteryGift_GetByIdx(MysteryGiftSave* mg, int index);
```

Comment:
Index instead of Idx

Reply:
What's the real advantage of spelling out "index" here? "Idx" is universally understood to mean "Index".

Reply:
style agreed idx bad

Reply:
reddo: "I consent to style"
donnel: "I consent to style"
pika: "I don't"

is there someone you forgot to ask?

Reply:
Idgaf as long as it's readable, Idx is still better than I